test_main.py

```diff
@@ -0,0 +1,33 @@
+"""
+This is the test !!!
+"""
+import unittest
+from app import get_guess_computer, get_guess_player, mainFunction
+from unittest.mock import patch, MagicMock, Mock
+
+class TestMainCode(unittest.TestCase):
+    
+    def test_get_computer(self):
+            result = get_guess_computer()
+            self.assertGreaterEqual(result, 1)
+            self.assertLessEqual(result, 100)    
+
+    @patch('builtins.input', return_value=22)
```

Comment:
Recuerda que input debe devolver un string

```suggestion
    @patch('builtins.input', return_value='22')
```